js/db.js

```diff
@@ -756,6 +756,93 @@ define(["dao", "globals", "lib/bluebird", "lib/davis", "lib/underscore", "util/h
                     playerFeatStore.createIndex("tid", "tid", {unique: false});
                 }());
             }
+            if(event.oldVersion <= 13) {
```

Comment:
I just pushed some new changes that already use this version. So this pull request should merge with master and then this version will have to be updated to 14.

@briansd9 are you still planning to update this pull request? No rush, just wondering.